appengine/users/src/main/java/com/example/appengine/users/UsersServlet.java

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START users_API_example]
+package com.example.appengine.users;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+public class UsersServlet extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        UserService userService = UserServiceFactory.getUserService();
+
+        String thisURL = req.getRequestURI();
+
+        resp.setContentType("text/html");
+	if (req.getUserPrincipal() != null) {
+            resp.getWriter().println("<p>Hello, " +
+                                     req.getUserPrincipal().getName() +
```

Comment:
The `+` should go on the continuation line as well.

To run checkstyle, you should be able to run it and the tests with `mvn clean verify`. To skip the tests and just build & check `mvn clean verify -DskipTests`.